### PR TITLE
doc: PCD library: link fix

### PR DESCRIPTION
**Pull request by peknis**
Fixing a reported wrong link.

### CODE REVIEW

File: doc/nrf/libraries/dfu/pcd.rst [hunk 1]
```diff
@@ -28,8 +28,8 @@ This is done to avoid compromised code from being able to perform updates of the
 The network core uses the PCD library to look for instructions on where to find the updates.
 Once an update instruction is found, this library is used to transfer the firmware update image.
 
-On the application core, the PCD library is used by the :doc:`mcuboot:index-ncs` sample.
-On the network core, the PCD library is used by the :ref:`nc_bootloader` sample.
+On the application core, the this library is used when having :ref:`MCUboot as an immutable bootloader <ug_bootloader_adding_upgradable_mcuboot>`.
```
Comment (hieuv) [inline doc/nrf/libraries/dfu/pcd.rst:31]: @hakonfam Is PCD only used by MCUboot when it is the immutable bootloader? When NSIB is the immutable one, PCD is still used by MCUboot, right?

File: doc/nrf/libraries/dfu/pcd.rst [hunk 2]
```diff
@@ -28,8 +28,8 @@ This is done to avoid compromised code from being able to perform updates of the
 The network core uses the PCD library to look for instructions on where to find the updates.
 Once an update instruction is found, this library is used to transfer the firmware update image.
 
-On the application core, the PCD library is used by the :doc:`mcuboot:index-ncs` sample.
-On the network core, the PCD library is used by the :ref:`nc_bootloader` sample.
```
Comment (hieuv) [inline doc/nrf/libraries/dfu/pcd.rst:32]: @peknis I wonder if we should call the Network Core Bootloader a "sample." It's located in samples, but it is meant to be used as-is, and is being used as-is.

File: doc/nrf/libraries/dfu/pcd.rst [hunk 3]
```diff
@@ -28,8 +28,8 @@ This is done to avoid compromised code from being able to perform updates of the
...
+On the application core, the this library is used when having :ref:`MCUboot as an immutable bootloader <ug_bootloader_adding_upgradable_mcuboot>`.
+On the network core, the library is used by the :ref:`nc_bootloader` sample.
```
[the remark this answers is not in the document]
Comment (peknis) [reply doc/nrf/libraries/dfu/pcd.rst:32]: Sorry, cannot really answer this.

[the remark this answers is not in the document]
Comment (sigvartmh) [reply doc/nrf/libraries/dfu/pcd.rst:32]: It's provided as a sample in-case you want to modify it to add some special functionality. I don't know if you actually can override the location of it in the build system and have your own out of tree version of this.

File: doc/nrf/libraries/dfu/pcd.rst (hunk 1 above)
Comment (hieuv) [reply doc/nrf/libraries/dfu/pcd.rst]: I understand that PCD is used by MCUboot and not NSIB. What I mean is that the doc is saying:
_[PCD] is used when having MCUboot as an immutable bootloader_

If I understand correctly, PCD is used whether MCUboot is immutable or not (i.e. whether NSIB is the immutable BL or not).

**Comment by peknis** [comment]
We are a bit stuck with this. @sigvartmh and @hakonfam, please suggest better wording.

**Review by sigvartmh — APPROVED** [review]

File: doc/nrf/libraries/dfu/pcd.rst (hunk 1 above)
Comment (sigvartmh) [reply doc/nrf/libraries/dfu/pcd.rst]: That's correct. PCD is used in MCUBoot in either case.

Comment (hieuv) [reply doc/nrf/libraries/dfu/pcd.rst]: Then I propose this instead:
<code>On the application core, this library is used by :ref:\`MCUboot <ug_bootloader_adding_upgradable_mcuboot>\`.</code>

File: doc/nrf/libraries/dfu/pcd.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (hieuv) [reply doc/nrf/libraries/dfu/pcd.rst:32]: This bit has always been tricky. Using "sample" is OK, I guess.
I haven't seen b0n location being overridden yet, but I have seen it being done for hci_rpmsg location. It's most likely possible.

**Pull request merged**